### PR TITLE
Fix single char typo in spec.md

### PR DESCRIPTION
**Pull request by econtal**
Missing backtick messed up the Markdown formatting in "entrypoint"

### CODE REVIEW

**Review by EricHripko — APPROVED** [review]
Looks good ✅ Thank you for this contribution @econtal 👍 Could you please sign your commit off to make sure that the PR checks are passing?

**Comment by EricHripko** [comment]
cc @kohidave

**Comment by econtal** [comment]
> Could you please sign your commit off to make sure that the PR checks are passing?

Oops, I just fixed via force-push!

**Comment by EricHripko** [comment]
Thanks 🙂 Once another maintainer approves, we'll be able to land 👍

**Review by ndeloof — APPROVED** [review]

**Pull request merged**
